data retrieval/src/main/java/de/tinf15b4/kino/retrieval/scraper/AbstractCinemaScraper.java

```diff
@@ -46,33 +48,33 @@ public AbstractCinemaScraper() {
 
     public void scrape() {
         logger = getLogger();
-        logger.info(String.format("Initializing Webdriver for scraper: [%s]", cinema.getName()));
+        logger.info("Initializing Webdriver for scraper: [%s]", cinema.getName());
```

Comment:
As did I. But apparently this only evaluates the string and all parameters if needed. If the logging level does not show the message it doesnt eval it. So actually its pretty nice :D